### PR TITLE
Adop 101 eligibility

### PR DESCRIPTION
**Pull request by dilipsamrahmcts**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

https://tools.hmcts.net/jira/browse/ADOP-183

Added new eligibility screens plus unit tests

**Does this PR introduce a breaking change?** (check one with "x")

- [ ] Yes
- [ ] No

### CODE REVIEW

**Review by nraghu32 — DISMISSED** [review]

**Review by abidfs — CHANGES_REQUESTED** [review]
minor changes

File: src/main/steps/eligibility/married/content.test.ts [hunk 1]
```diff
@@ -1,17 +1,61 @@
...
+import { isFieldFilledIn } from '../../../app/form/validation';
+import { CommonContent, generatePageContent } from '../../common/common.content';
 
 import { generateContent } from './content';
 
+jest.mock('../../../app/form/validation');
+
+/* eslint-disable @typescript-eslint/ban-types */
+
 describe('eligibility content', () => {
```
Comment (abidfs) [inline src/main/steps/eligibility/married/content.test.ts:12]:
```suggestion
describe('married content', () => {
```

File: src/main/steps/eligibility/under-18/content.test.ts [hunk 2]
```diff
@@ -1,21 +1,64 @@
...
+import { FormContent, FormFields, FormInput, FormOptions } from '../../../app/form/Form';
+import { isFieldFilledIn } from '../../../app/form/validation';
+import { CommonContent, generatePageContent } from '../../common/common.content';
 
 import { generateContent } from './content';
 
+jest.mock('../../../app/form/validation');
+
+/* eslint-disable @typescript-eslint/ban-types */
 describe('eligibility content', () => {
```
Comment (abidfs) [inline src/main/steps/eligibility/under-18/content.test.ts:11]:
```suggestion
describe('under-18 content', () => {
```

File: src/main/steps/eligibility/under-21/content.test.ts [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+import { isFieldFilledIn } from '../../../app/form/validation';
+import { CommonContent, generatePageContent } from '../../common/common.content';
+
+import { generateContent } from './content';
+
+jest.mock('../../../app/form/validation');
+
+/* eslint-disable @typescript-eslint/ban-types */
+
+describe('eligibility content', () => {
```
Comment (abidfs) [inline src/main/steps/eligibility/under-21/content.test.ts:12]:
```suggestion
describe('under-21 content', () => {
```

**Review by abidfs — CHANGES_REQUESTED** [review]
minor changes

**Review by abidfs — APPROVED** [review]

**Pull request merged**